recipes/metaclock/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "metaclock" %}
+{% set version = "1.0.0" %}
+
+
+
+package:
+  name:  "{{ name }}"
+  version:  "{{ version }}"
+
+source:
+  url: https://github.com/SegataLab/metaclock/archive/{{ version }}.tar.gz
+  sha256: 146e6e8b5c8d62dc958a41cfb5a53cae9cc57bbc1da82d4020b217aafc180429
+
+build:
+  noarch: python
```

Comment:
Can you add the entry points, (e.g. like here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pmx_biobb/meta.yaml#L17) similar to how they are specified in the setup.py?